spec-main/api-tray-spec.ts

```diff
@@ -26,18 +28,16 @@ describe('tray module', () => {
 
   describe('tray.destroy()', () => {
     it('destroys a tray', () => {
-      expect(tray.isDestroyed()).to.be.false()
+      expect(tray.isDestroyed()).to.equal(false)
```

Comment:
```suggestion
      expect(tray.isDestroyed()).to.be.false('tray is destroyed')
```

this helps a bit when an error is reported, showing what the boolean represents in addition to the info that it's not what was expected.